src/inc/api/APISendProgress.class.php

```diff
@@ -229,6 +229,14 @@ public function execute($QUERY = array()) {
           $qF3 = new QueryFilter(Hash::IS_CRACKED, 0, "=");
           $hashes = Factory::getHashFactory()->filter([Factory::FILTER => [$qF1, $qF2, $qF3]]);
           if (sizeof($hashes) == 0) {
+            //This can happen if agent rebuild the hash incorrectly
+            //Log the skipped hash so that admin can spot this false negative
+            $logMessage = "Hash has been cracked but skipped! This happened while cracking hashlist with Id: "
```

Comment:
We should make it consistent with the writing of id/Id, I would propose to use the uppercase `ID` writing. Also the `Does` should be written lowercase, and at the end of the log message I would add a dot.